build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts

```diff
@@ -633,6 +633,23 @@ class ApiLevelReader {
 			addUniqueNamedExportsToMap(exports.alpha, memberData, ApiLevel.internal);
 		} else {
 			addUniqueNamedExportsToMap(exports.legacy, memberData, ApiLevel.legacy);
+			if (exports.legacyAlpha.length > 0) {
+				// @alpha APIs have been mapped to both /alpha and /legacy paths.
+				// Later @legacy tag was added explicitly.
+				// Check for a /legacy/alpha export to map @legacy & @alpha as legacy/alpha.
+				const legacyAlphaExport =
+					this.tempSource
+						.addImportDeclaration({
+							moduleSpecifier: `${packageName}/legacyAlpha`,
```

Comment:
i'd like this to be `legacy/alpha`, but it breaks node10 compat, which needs to map the export name to a filename. i also considered `legacy-alpha`, not sure what the best solution is.